### PR TITLE
Bulk domain transfer flow: Update copy to recommended strings.

### PR DESCRIPTION
**Pull request by allilevine**
Related to https://github.com/Automattic/dotcom-forge/issues/2987

## Proposed Changes

* Update copy.

Before | After
------ | ------
Screen Shot 2023-07-10 at 3 36 00 PM | Screen Shot 2023-07-10 at 3 35 02 PM
Screen Shot 2023-07-10 at 3 35 50 PM | Screen Shot 2023-07-11 at 12 05 26 PM

## Testing Instructions

* Check that copy is updated on Intro and Complete pages in the bulk domain transfer flow.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by allilevine** [comment]
@alshakero Can you confirm if we email the user when their domain transfer is complete? Otherwise this change may not make sense:

> We've got it from here! We'll let you know when your newly transferred domain is ready to use.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8360694

Thank you @allilevine for including a screenshot in the description! This is really helpful for our translators.

**Comment by provenself** [comment]
On the Complete page, consider this a vote to reinstate the exclamation point on the heading text: **Congrats on your domain transfers!** (We could then lose the exclamation point from `We'll let you know when your newly transferred domains are ready to use!` and just use a period: `We'll let you know when your newly transferred domains are ready to use.`)

Otherwise looks great! 👍

File: client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-complete/index.tsx
Comment (provenself) [inline client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-complete/index.tsx:1]: Dropped this in the main thread, but can we change the second sentence from an exclamation to a period?

**Review by provenself — COMMENTED** [review]
On the subheading for the Complete page, can we change the second sentence from an exclamation to a period? 

Would read:
`We've got it from here! We'll let you know when your newly transferred domains are ready to use.`

Also, would love to reinstate the exclamation point on the heading text: `Congrats on your domain transfers!`
Otherwise looks great! 👍

(Sorry for duplicating this everywhere—my first time doing an actual GH review!)

**Comment by allilevine** [comment]
@provenself Thanks, I committed both changes!

Screen Shot 2023-07-11 at 12 05 26 PM

**Comment by allilevine** [comment]
> @alshakero Can you confirm if we email the user when their domain transfer is complete? Otherwise this change may not make sense:
> 
> > We've got it from here! We'll let you know when your newly transferred domain is ready to use.

Nm I found answers in our docs: https://github.com/Automattic/dotcom-forge/issues/2987#issuecomment-1631512304

**Review by alshakero — APPROVED** [review]
Looks great. Sorry for the late review! I'll merge and deploy.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8360694).

**Comment by allilevine** [comment]
> Looks great. Sorry for the late review! I'll merge and deploy.

Thank you @alshakero!

**Pull request merged**